src/main/java/com/sampleweb/pages/GroceryPage.java

```diff
@@ -0,0 +1,94 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.List;
+
+public class GroceryPage extends HomePage {
+
+    public static String PATH = "https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dgrocery&field-keywords=";
+    public static String HOMEPAGE_PATH = "https://www.amazon.co.uk/";
+
+    public String title = "Grocery";
+    public String pageTitle = "//h1[text()=\"Grocery\"]";
+    public String categorySearchDropdown = "//select[@id='searchDropdownBox']";
```

Comment:
Should be private since this will not be accessed from anywhere else. Same with the ones below.